client/websocket.go

```diff
@@ -184,22 +208,93 @@ func (c *Client) handleWSMsg(msg ws.Message) error {
 				return errCallEnded
 			}
 		case wsEventCallJobState:
+			callID, _ := ev.GetData()["callID"].(string)
+			if callID != c.cfg.ChannelID {
+				// Ignore if the event is not for the current call/channel.
+				return nil
+			}
+
 			data, ok := ev.GetData()["jobState"].(map[string]any)
 			if !ok {
 				return fmt.Errorf("invalid recording state")
 			}
 			var jobState CallJobState
 			jobState.FromMap(data)
-			c.emit(WSCallJobState, jobState)
+			c.emit(WSCallJobStateEvent, jobState)
 
 			// Below is deprecated as of v0.14.0, kept for compatibility with earlier versions
 			// of transcriber
 			if jobState.Type == "recording" {
-				c.emit(WSCallRecordingState, jobState)
+				c.emit(WSCallRecordingStateEvent, jobState)
```

Comment:
Yes although whenever we update our transcriber to use this we'll have to change a couple of things since I made some breaking changes to the client API.